### ISSUE TITLE
Foreach => Yield From Conversion Breaks Nested Loops

### ISSUE THREAD

**Issue by kiler129** [kalessil/phpinspectionsea#1256]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) v3.0.12 |
| Language level | 7.2                        |

Current behaviour
--
Consider the following code:
```php
<?php

class test
{
    private $foo = [
        ['a', 'b', 'c'],
        ['d', 'e', 'f'],
    ];

    function loop()
    {
        foreach ($this->foo as $c) {
            foreach ($c as $i) {
                yield $i;
            }
        }
    }
}
```

The first thing suggested is correct:
![image](https://user-images.githubusercontent.com/1227834/55080406-204b3300-506c-11e9-93f6-fc425b6d37bb.png)

↓↓↓

![image](https://user-images.githubusercontent.com/1227834/55080439-29d49b00-506c-11e9-82ba-711d62632cd2.png)

However, the next suggestion given is wrong and after applying breaks the code:
![image](https://user-images.githubusercontent.com/1227834/55080532-54265880-506c-11e9-8b98-50c5084e8b1d.png)

↓↓↓

![image](https://user-images.githubusercontent.com/1227834/55080642-74eeae00-506c-11e9-9e39-c1995bd64e35.png)

Expected behaviour
--
I don't believe you can chain `yield from`s (e.g. `yield from (yield from $this->foo);` causes a fatal error), so I think the expected behavior will be to not suggest such replacement in the second case ;)

Environment details
---
```
PhpStorm 2019.1
Build #PS-191.6183.55, built on March 20, 2019
Licensed to *****
Subscription is active until April 19, 2019
JRE: 1.8.0_202-release-1483-b39 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-46-generic
```

**Comment by funivan** [comment]
@kiler129 can you check this issue https://github.com/kalessil/phpinspectionsea/issues/1167 I think they are about the same bug)

**Comment by kiler129** [comment]
@funivan Indeed looks like the same problem (but here it's directly showing two nested foreaches).

**Comment by kalessil** [comment]
Duplicates #1167

**Comment by kalessil** [comment]
Fixed!
